utils/pub/pubDates.ts

```diff
@@ -77,8 +77,8 @@ export const getPubCopyrightYear = (
 		if (metadata) {
 			const { copyrightYear, date, publicationDate } = metadata;
 			const dateSource = copyrightYear || date || publicationDate;
-			if (dateSource) {
-				return dateFormat(dateSource, 'yyyy');
+			if (dateSource && new Date(dateSource).getDate()) {
```

Comment:
Could you leave a comment about what's happening here?

Reply:
In the code or here?

Long story short: this fixes two bugs. 

1. Currently, if you add something in that meta field that is not interpretable as a date, it will break any Pubs in the collection. That's what checking `new Date(...)` in the conditional prevents. I've also added some form validation to prevent it, so we could probably remove this. 

2. There's a timezone-related bug where, if your local time is > UTC, the copyright will always be displayed as the year prior to what's actually entered, because it stores it as midnight UTC on January 1. `Date.UTC(...)` is a bit of a hack to make it always display the year as entered, which is the intended behavior independent of timezone. It does this by asserting that the date is actually February 1 UTC.

Reply:
There are functions called `isValidDate` and `getLocalDateMatchingUtcCalendarDate` in `utils/dates.ts` that I think you might be able to use here. It might be nice to modify `isValidDate` to take `string | Date` and do the cast to `Date` internally if necessary.